packages/localize/src/localize-theia.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) 2017 Ericson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+"use strict";
+const { exec } = require('child_process');
+const fs = require('fs');
+
+/*
+* Command to create the i18n folder for globalization and the localise file to use in theia.
+* i.e.:  localize-theia.ts <package>
+  ex:  localize-theia.ts core
+*/
+function createThis() {
+    if (process.argv.length > 2) {
+        // Read the first additional argument passed to the program
+        const myPackage = process.argv[2];
+        const localizeFile = "localize.ts";
+        const browserPath = "./src/browser/";
+        const i18nPath = "./i18n";
+
+        const i18nExampleFile = i18nPath + "/en.json"
+
+        let packagePath = "";
+        let content = copyRight(); // insert the file copyright to start
+
+        // Set the content of the file
+        packagePath = browserPath;
```

Comment:
too many path variables there...
It could be just packagePath = ./src/browser/

Reply:
Done